kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/flightsearch/Application.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.good.patterns.challenges.flightsearch;
+
+public class Application {
+    public static void main(String[] ars) {
+
+        FlightRequestRetriever flightRequestRetriever = new FlightRequestRetriever();
+        FlightRequest allFlights = flightRequestRetriever.retrieve();
+
+        FlightSearchEngine flightSearchEngine = new FlightSearchEngine(allFlights.getAllFlights());
+        flightSearchEngine.searchByFlightArrival("Warszawa");
```

Comment:
Mam jedną uwagę na początek. Twoja wyszukiwarka powinna zwracać obiekt reprezentujący wynik wyszukiwania, i jeśli gdziekolwiek chcesz użyć `System.out.println` aby wypisać to na konsolę - to w tej klasie :) 
Wieczorem wrócę z bardziej szczegółowym review.

Reply:
Uwaga niezrealizowana - `System.out.println` tutaj (i najlepiej tylko tutaj).